cargo-creusot/src/why3find_wrapper.rs

```diff
@@ -27,8 +27,8 @@ fn why3find_json_exists() -> bool {
     Path::new("why3find.json").exists()
 }
 
-fn raw_config(args: &Vec<String>, why3_config_file: &PathBuf) -> Result<()> {
-    let mut why3find = Command::new("why3find");
+fn raw_config(args: &Vec<String>, why3find: &PathBuf, why3_config_file: &PathBuf) -> Result<()> {
```

Comment:
Why not pass the full `&creusot_setup::CreusotFlags`, like in `raw_prove` ?

Reply:
I think I'm going to do that for consistency. At first I was just not sure if that's the right way to use `CreusotFlags` and `status_for_creusot` because of their not very descriptive and probably outdated names.